src/app/pages/send/send-crypto-asset-form/family/stacks/hooks/use-stacks-broadcast-transaction.tsx

```diff
@@ -62,12 +70,16 @@ export function useStacksBroadcastTransaction(unsignedTx: string) {
             txid: txId || '',
           },
           txId,
-          ...formReviewTxSummary(signedTx),
+          ...formReviewTxSummary(signedTx, token),
         },
       };
     }
 
-    function formReviewTxSummary(tx: StacksTransaction) {
+    function formReviewTxSummary(tx: StacksTransaction, symbol = 'STX', decimals = 6) {
```

Comment:
Looking at some of these functions, do they need to be located in `use-stacks-broadcast-transaction`? Could they be pulled out into their own hook to handle the summary itself to reduce the complexity here? Another file, `use-transaction-summary`?